stylelint.config.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import('stylelint').Config} */
+const config = {
+  extends: ['stylelint-config-upleveled'],
+};
+
+export default config;
```

Comment:
seems like another problem caused by the broken project type detection in `eslint-config-upleveled` - Stylelint should only be installed in CRA and Next.js apps:

- https://github.com/upleveled/eslint-config-upleveled/blob/6fafffc0012b70e1a54aabdfca000a1e083e1ce4/bin/install.js#L110-L121

Reply:
Was fixed in this [ESLint PR](https://github.com/upleveled/eslint-config-upleveled/pull/398), and updated with this commit in this PR [f2c33e6](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/4/commits/f2c33e6e7317cb16183dd16a79f1845338d6cfbe)